script/update-build-tomcat.sh

```diff
@@ -119,4 +152,4 @@ chmod -R g+w *
 # Give execute to Others for directories only
 find /var/www/html/calcentral/ -type d -execdir chmod o+x {} \;
 
-exit 0
+###exit 0
```

Comment:
Should the line be removed?

Reply:
Yes, you can remove that line. I will revisit when better error handling is incorporated to the script. For now it is just a modified version of the existing one, and will suffice.